Sources/Shared/Models/VIMVideo.m

```diff
@@ -80,7 +80,8 @@ - (NSDictionary *)getObjectMapping {
         @"play": @"playRepresentation",
         @"review_page": @"reviewPage",
         @"file_transfer": @"fileTransfer",
-        @"app": @"sourceClientApp"
+        @"app": @"sourceClientApp",
+        @"publish_to_social": @"publishToSocialJob"
```

Comment:
See below comment. If you will rename to `publishToSocial`, then this should be renamed to `publishToSocial` also